packages/code-du-travail-api/src/routes/enterprises/index.ts

```diff
@@ -13,7 +13,7 @@ const index = `${ES_INDEX_PREFIX}-${CDTN_ADMIN_VERSION}_${DOCUMENTS}`;
 
 const router = new Router({ prefix: API_BASE_URL });
 const ENTERPRISE_API_URL =
-  "https://search-recherche-entreprises.fabrique.social.gouv.fr/api/v1";
+  "https://api.recherche-entreprises.fabrique.social.gouv.fr/api/v1";
```

Comment:
Apply this change : https://github.com/SocialGouv/code-du-travail-numerique/pull/4259